lib/Cleantalk/ApbctWP/Firewall/SFW.php

```diff
@@ -190,6 +190,10 @@ public function check()
                     );
 
                     $test_status = (int)$db_result['status'];
+
+                    if ($text_status === 'PASS_SFW__BY_WHITELIST') {
+                        break;
+                    }
```

Comment:
Look at row 174. This statement is never reached because of the break. Isn't it?

Reply:
At row 174 break need for out of switch condition, at row 195 break need for out of the loop.